docs/buildh.md

```diff
@@ -35,24 +53,24 @@ optional arguments:
   -o OUT, --out OUT     Output base name for storing order parameters.
                         Extention ".out" will be automatically added. Default
                         name is OP_buildH.out.
+  -b BEGIN, --begin BEGIN
+                        The first frame (ps) to read from the trajectory.
+  -e END, --end END     The last frame (ps) to read from the trajectory.
+  -pi PICKLE, --pickle PICKLE
+                        Output pickle filename. The structure pickled is a dictonnary containing for each Order parameter,
+                        the value of each lipid and each frame as a matric
+
+The list of supported lipids (-l option) are: CHARMM_POPC, Berger_POPC, Berger_PLA, Berger_POP.
 ```
 
-The program needs two mandatory files (present in this repo):
-- `dic_lipids.py` (option `-l`, present in this repo) ;
-- `order_parameter_definitions_MODEL_Berger_POPC.def` (option `-d`, present in this repo).
-
 
-#### dic_lipids.py
-
-This file is used as a module and contains a list of dictionaries based on the type of lipids (POPC,DOPC,...) and the force field (Berger, GROMOS, etc). The chosen lipid/FF is passed to `buildH` with `-l` option (e.g. `-l Berger_POPC`).
-The dictionary is a list of carbon atoms from which the hydrogens are built.
-For each carbon atom, there is the type of bonds (CH3, CH2, etc) and the 2 (or 3) others atoms needed for the reconstruction (see Algorithm).
-So far, the file contains Berger POPC and CHARMM36 POPC (this latter is used for validation only, since it is an all-atom force field). It will be updated in the future with some other united-atom force fields and other lipid types models.
```

Comment:
I agree we have to remove this section on `dic_lipids.py` because this file no longer exists. But we should plan to replace it by the description of json files somewhere!